engine/src/operators/GroupBy.cpp

```diff
@@ -533,6 +550,15 @@ std::unique_ptr<ral::frame::BlazingTable> compute_aggregations_with_groupby(
 						if (temp->size() != 0) {
 							cudf::experimental::fill_in_place(mview_temp, 0, temp->size(), *scalar);
 						}
+						aggregation_inputs_scope_holder.emplace_back(std::move(temp));
+						aggregation_input = aggregation_inputs_scope_holder.back()->view();
+					}else if(aggregation_types[i] == AggregateKind::SUM0){
+
+						std::unique_ptr<cudf::scalar> scalar = cudf::make_numeric_scalar(table.view().column(column_index).type());
+						std::unique_ptr<cudf::column> temp = cudf::experimental::replace_nulls(table.view().column(column_index), *scalar );
+						auto numeric_s = static_cast< cudf::experimental::scalar_type_t<int8_t>* >(scalar.get());
```

Comment:
I dont think this will work. The data type of your scalar is not int_8 its of type table.view().column(column_index).type()  . You may need to use the type_dispatcher here or something.  I know 0 is basically 0 regardless of the data type, but given different widths, i am not sure if the behaviour is guaranteed to do what you want.